services/src/floodlight-modules/src/main/java/org/openkilda/floodlight/command/ping/PingCommand.java

```diff
@@ -31,19 +31,21 @@
 import java.util.UUID;
 
 abstract class PingCommand extends Command {
-    protected static Logger logPing = LoggerFactory.getLogger("open-kilda.flows.PING");
+    protected static final Logger logPing = LoggerFactory.getLogger("open-kilda.flows.PING");
 
     private final KafkaMessageProducer kafkaProducer;
-    private final ConfigService configService;
     private final PingService pingService;
 
+    private final KafkaTopicsConfig topics;
+
     PingCommand(CommandContext context) {
         super(context);
 
         FloodlightModuleContext moduleContext = getContext().getModuleContext();
-        configService = moduleContext.getServiceImpl(ConfigService.class);
         pingService = moduleContext.getServiceImpl(PingService.class);
         kafkaProducer = moduleContext.getServiceImpl(KafkaMessageProducer.class);
+
+        topics = moduleContext.getServiceImpl(KafkaMessageProducer.class).getTopics();
```

Comment:
kafkaProducer is already obtained a line above, no sense to repeat.

Reply:
Nice catch, thanks. Fixed.